tests/integration_tests/style/test_python.py

```diff
@@ -27,7 +27,7 @@ def test_python_style():
     python_files = utils.get_files_from(
         find_path="..",
         pattern="*.py",
-        exclude_names=["build"])
+        exclude_names=["build", ".kernel"])
```

Comment:
Not in the scope of the rebase, but I am wondering if we can download the kernel within the container, outside the firecracker workspace directory. Have you tried it?